src/mbgl/renderer/render_tile.cpp

```diff
@@ -174,12 +175,13 @@ void RenderTile::finishRender(PaintParameters& parameters) const {
             *debugBucket->indexBuffer,
             debugBucket->segments,
             program.computeAllUniformValues(DebugProgram::LayoutUniformValues{uniforms::matrix::Value(matrix),
-                                                                              uniforms::color::Value(Color::black())},
+                                                                              uniforms::color::Value(Color::black()),
+                                                                              uniforms::overlay_scale::Value(1.0f)},
                                             paintAttributeData,
                                             properties,
                                             parameters.state.getZoom()),
             allAttributeBindings,
-            DebugProgram::TextureBindings{},
+            DebugProgram::TextureBindings{textures::image::Value{debugBucket->texture->getResource()}},
```

Comment:
is debug texture guaranteed to exist at this point?

Reply:
yeah, the pass is `prepare -> upload -> render`